static/js/Sensor.js

```diff
@@ -85,3 +88,37 @@ class TapeLine{
     }
   }
 }
+
+class LimitSwitch{
+  constructor(robot) {
+    this.robot = robot;
+    this.switch0 = false;
+    this.switch1 = false;
+    this.leeway = 1;
```

Comment:
Can you add a comment about this being in unit of inches (i.e. the units used everywhere in robot.js)?